tests/processes/ProcessingTest.py

```diff
@@ -636,6 +603,46 @@ def check_prerequisites(cls):
                                   console_printer=self.console_printer)
         self.assertGreater(len(cache.data), 0)
 
+    def test_global_instantiation(self):
+        class TestThreeBear(Bear):
+
+            def __init__(self, file_dict, section, queue, timeout=0.1,
+                         debugger=False):
+                raise RuntimeError
+
+        class TestTwoBear(Bear):
+
+            BEAR_DEPS = {TestThreeBear}
+
+            def __init__(self, file_dict, section, queue, timeout=0.1,
+                         debugger=False):
+                raise RuntimeError
+
+        class TestOneBear(Bear):
+
+            BEAR_DEPS = {TestTwoBear}
+
+            def __init__(self, file_dict, section, queue, timeout=0.1,
+                         debugger=False):
+                Bear.__init__(self, section, queue, timeout, debugger)
```

Comment:
redundant `__init__`

Reply:
Actually `__init__` is not redundant here as I am checking whether Bear gets initialized correctly without any error, If I remove the `__init__` then it will throw error

`TypeError: section must be an instance of one of (<class 'coalib.settings.Section.Section'>,) (provided value: {})`

Reply:
yeah, because you pass a file-dict where you should pass the section :) You just changed the signature, and forgot to use `file_dict` as well ;)